hosts/src/bin/cmdprocessor.rs

```diff
@@ -56,8 +56,10 @@ fn handle_command(cmd: CliCommand) -> Result<(), Box<dyn ::std::error::Error>> {
 fn main() -> Result<(), Box<dyn ::std::error::Error>> {
     let args = Cli::from_args();
     let cmd = args.command;
-    env_logger::init();
-
+    match env_logger::try_init() {
```

Comment:
Here you don't need to do the match, you can use the "ignore a result" syntax like this:

`let _ = env_logger::try_init()`

You can also clean up how the module host displays logs with the following:
```
    let _ = env_logger::builder().format_module_path(false).try_init();
```